### PR TITLE
End distributions when empty() is called

### PR DESCRIPTION
**Pull request by vanruch**

### CODE REVIEW

File: contracts/truefi/distributors/LinearTrueDistributor.sol [hunk 1]
```diff
@@ -126,10 +126,13 @@ contract LinearTrueDistributor is ITrueDistributor, Ownable {
 
     /**
      * @dev Withdraw funds (for instance if owner decides to create a new distribution)
-     * Distributes remaining funds before withdrwaing
+     * Distributes remaining funds before withdrawing
+     * Ends current distribution
      */
     function empty() public override onlyOwner {
         distribute();
+        distributed = 0;
```
Comment (barrutko) [inline contracts/truefi/distributors/LinearTrueDistributor.sol:134]: Should distributed be set to 0?

File: contracts/truefi/distributors/ArbitraryDistributor.sol [hunk 2]
```diff
@@ -67,12 +69,15 @@ contract ArbitraryDistributor is IArbitraryDistributor, Ownable {
     function distribute(uint256 _amount) public override onlyBeneficiary {
         remaining = remaining.sub(_amount);
         require(trustToken.transfer(msg.sender, _amount));
+
+        emit Distributed(_amount);
```
Comment (barrutko) [inline contracts/truefi/distributors/ArbitraryDistributor.sol:73]: 👍

**Review by barrutko — APPROVED** [review]
`distributed = 0;` looks a bit confusing, but apart from that looks fine 👌

File: contracts/truefi/distributors/LinearTrueDistributor.sol (hunk 1 above)
Comment (vanruch) [reply contracts/truefi/distributors/LinearTrueDistributor.sol:134]: If it's not, this line would throw `totalAmount.sub(distributed);`

**Pull request merged**